.github/workflows/release-assets.yml

```
@@ -0,0 +1,115 @@
+        with:
+          go-version-file: go.mod
+          cache: true
+
+      # build-dist-manifests renders the chart via `helm template`.
+      - name: Install Helm
+        uses: azure/setup-helm@v4
+        with:
+          version: 'v3.16.4'
+
+      - name: Resolve release tag
+        run: echo "RELEASE_TAG=${{ github.ref_name }}" >> $GITHUB_ENV
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:46: Double quote to prevent globbing and word splitting

(shellcheck)